ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -1,2 +1,28 @@
 """Includes Mindspore Frontend functions listed in the TODO list
 https://github.com/unifyai/ivy/issues/14951."""
+
+# local
+import ivy
+from ivy.func_wrapper import with_supported_dtypes
+from ivy.functional.frontends.paddle.func_wrapper import to_ivy_arrays_and_back
+
+
+@with_supported_dtypes({"2.0 and below": ("float16", "float32")}, "mindspore")
+@to_ivy_arrays_and_back
+def softsign(x):
+    """
```

Comment:
I don't think we need to include docstrings in the frontend as users wont really interact with these functions.

Reply:
Alright, apologies for the impractical implementation, I have removed the docstrings in my latest commit